lms/djangoapps/verified_track_content/models.py

```diff
@@ -3,8 +3,66 @@
 """
 from django.db import models
 from django.utils.translation import ugettext_lazy
+from django.dispatch import receiver
+from django.db.models.signals import post_save, pre_save
 
 from xmodule_django.models import CourseKeyField
+from student.models import CourseEnrollment
+from courseware.courses import get_course_by_id
+
+from verified_track_content.tasks import sync_cohort_with_mode, VERIFIED_COHORT_NAME
+from openedx.core.djangoapps.course_groups.cohorts import (
+    get_course_cohorts, CourseCohort, is_course_cohorted
+)
+
+import logging
+
+log = logging.getLogger(__name__)
+
+VERIFIED_COHORT_NAME = VERIFIED_COHORT_NAME
```

Comment:
I did this so that the test code could import VERIFIED_COHORT_NAME from models instead of tasks.

In reality it doesn't really matter because we will move away from a hardcoded cohort name in the next story. So I will just remove this and make the test depend on on tasks also (which actually I see that it already does...).